recipes-rubygems/rubygems-nokogiri_1.11.5.bb

```diff
@@ -6,7 +6,7 @@ HOMEPAGE = "https://nokogiri.org"
 LICENSE = "MIT"
 LIC_FILES_CHKSUM = "file://LICENSE-DEPENDENCIES.md;md5=2fcfc4989eb71e592d2e85d26d7fa91c"
 
-EXTRA_DEPENDS += "\
+EXTRA_DEPENDS_append += " \
     libxml2 \
```

Comment:
potential issue: Variable 'LICENSE' is set by rubygems-nokogiri_1.11.5.bb,rubygems-nokogiri_1.12.3.bb